src/services/quota/index.tsx

```diff
@@ -142,6 +149,8 @@ export const liveGetQuota = async (
     }
     if (e.message === "User is not eligible") {
       throw new NotEligibleError(e.message);
+    } else if (e.message === "Invalid authentication token provided") {
+      throw new AuthenticationError(e.message);
```

Comment:
This error will occur whenever we try to use an invalid auth token on a protected endpoint, shall we move it to be inside the `fetchWithValidator` so we can ensure that for every protected endpoint, we could. get this auth error?

Reply:
sounds good!